src/components/ColorPicker/ColorPickerItem/ColorPickerItem.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import styles from './ColorPickerItem.st.css';
+import { RadioButton } from 'wix-ui-core/radio-button';
+
+export interface ColorPickerItemProps {
+  onChange?(color: string): void;
+  color: string;
+  ariaLabel: string;
+  checked?: boolean;
+}
+
+interface ColorPickerItemDefaultProps {
+  checked: boolean;
+  color: string;
+  onChange(color: string): void;
+}
+
+/** ColorPickerItem */
+export class ColorPickerItem extends React.Component<ColorPickerItemProps> {
+  static displayName = 'ColorPickerItem';
+  static defaultProps: ColorPickerItemDefaultProps = {
+    checked: false,
+    color: 'white',
+    onChange: (color: string) => console.log(color),
+  };
+
+  getRadio = ({ color }: ColorPickerItemProps) => (
+    <div className={styles.radioOuter}>
+      <div className={styles.radioInner} style={{ backgroundColor: color }} />
+    </div>
+  );
+
+  render() {
+    const { ariaLabel, color, checked, onChange } = this.props;
+
+    return (
+      <div {...styles('root', { checked }, { ...this.props })}>
+        <RadioButton
```

Comment:
I think we can remove this `div` and pass the styles directly to the radioButton component.
In addition we need to import the `RadioButton` in the `st.css` file and `-st-extend: RadioButton` .

Reply:
done